src/main/scala/com/github/unchama/seichiassist/menus/stickmenu/SecondPage.scala

```diff
@@ -59,7 +64,7 @@ object SecondPage extends Menu {
     player: Player
   )(implicit environment: Environment): IO[MenuSlotLayout] = {
     import ConstantButtons._
-    val computations = ButtonComputations(player)
+    val computations = ButtonComputations(player)(environment.sharedInventoryAPI)
     import computations._
     import environment._
```

Comment:
```suggestion
    import environment._
    val computations = ButtonComputations(player)
    import computations._
```

`import environment`で変数の型のメンバーを現在のスコープに持ち込めるので、明示的な`implicit`への引数渡しは必要なくなります